rust/src/write.rs

```diff
@@ -732,6 +732,33 @@ impl<W: Write + Seek> Writer<W> {
         Ok(())
     }
 
+    /// Write a private record. If the record can be present in chunks then the `chunkable` flag
+    /// should be set to true.
+    ///
+    /// Private records must have an opcode >= 0x80.
+    pub fn write_private_record(
+        &mut self,
+        opcode: u8,
+        data: &[u8],
+        chunkable: bool,
```

Comment:
consider a two-value enum rather than a bool here?

Reply:
I've added `PrivateRecordKind::Chunkable`. Happy to change if you have a better name.